### PR TITLE
enable sgx for DCsv3 ubuntu18.04

### PR DESCRIPTION
**Pull request by Francis-Liu**
Signed-off-by: Francis Liu <feliu@microsoft.com>

### CODE REVIEW

File: Testscripts/Windows/VALIDATE-INTEL-SGX-DRIVER.ps1 [hunk 1]
```diff
@@ -31,7 +31,7 @@ function Main {
         }
     }
 
-    if (($ubuntuVersion -notmatch "Ubuntu 18.04") -and ($ubuntuVersion -notmatch "Ubuntu 16.04")) {
+    if ($ubuntuVersion -notmatch "Ubuntu 18.04") {
```
Comment (LiliDeng) [inline Testscripts/Windows/VALIDATE-INTEL-SGX-DRIVER.ps1:34]: does it mean all sgx cases only can run on ubuntu 18.04?

Comment (Francis-Liu) [reply Testscripts/Windows/VALIDATE-INTEL-SGX-DRIVER.ps1:34]: Yes, the latest version of OpenEnclave has abandoned 16.04 support.

Comment (LiliDeng) [reply Testscripts/Windows/VALIDATE-INTEL-SGX-DRIVER.ps1:34]: during the internal mail session, will it support higher ubuntu distro than 18.04 like ubuntu 20.04?

**Review by LiliDeng — DISMISSED** [review]

**Review by johnsongeorge-w — CHANGES_REQUESTED** [review]
Test is failing due to missing curl.h. Installing `libcurl4-openssl-dev` will resolve the issue.

**Review by johnsongeorge-w — APPROVED** [review]

File: XML/RegionAndStorageAccounts.xml [hunk 2]
```diff
@@ -1,111 +1,111 @@
...
+  <centralus>
+    <StandardStorage>eosgcentralus</StandardStorage>
+    <PremiumStorage>eosgcentralusx</PremiumStorage>
+  </centralus>
+  <eastus>
+    <StandardStorage>eosgeastus</StandardStorage>
+    <PremiumStorage>eosgeastusx</PremiumStorage>
+  </eastus>
+  <eastus2>
+    <StandardStorage>feliuuse2</StandardStorage>
```
Comment (LiliDeng) [inline XML/RegionAndStorageAccounts.xml:20]: @Francis-Liu @johnsongeorge-w Include this file change by mistake?

Comment (johnsongeorge-w) [reply XML/RegionAndStorageAccounts.xml:20]: https://github.com/microsoft/lisa/pull/1366 reverts this changes.

**Pull request merged**